### ISSUE TITLE
Broken Dynamic Trees With CaveBiomeAPI 

### ISSUE THREAD

**Issue by FetusEnchilada** [ItsBlackGear/CaveBiomeAPI#18]
Having both Dynamic Trees & CaveBiomesAPI installed some of the trees will generate broken and wierd. Tested with a clean forge 36.2.8 install and happens on both single player also multiplayer servers as well. 
![2021-11-02_14 26 15](https://user-images.githubusercontent.com/92892177/139955329-f1531aa2-237a-412e-9b31-a9308625f129.png)
![2021-11-02_14 25 58](https://user-images.githubusercontent.com/92892177/139955384-4fad47db-7049-4433-92b8-dac1ea8773cd.png)
![2021-11-02_14 24 54](https://user-images.githubusercontent.com/92892177/139955404-df6aa1e7-c931-4498-aef5-f4671c42612b.png)

**Comment by ItsBlackGear** [comment]
yeah :0 i've been aware of this issue and have to find what's causing it..

**Comment by FetusEnchilada** [comment]
good luck happy hunting godspeed

**Comment by ItsBlackGear** [comment]
a

**Comment by BastardNeko** [comment]
Any update on this?

**Comment by BastardNeko** [comment]
is this going to be fixed for 1.16.5?

**Comment by LIZARDLICORICE** [comment]
you find out any thing yet?

**Comment by ItsBlackGear** [comment]
nothing yet

**Comment by LIZARDLICORICE** [comment]
i believe it was said some where that the DT world gen And the CavebiomeAPI world gen conflict with each other for what ever reason
